src/main/java/ua/antibyte/cinema/service/inject/Inject.java

```diff
@@ -1,36 +1,58 @@
-package ua.antibyte.cinema.controller;
+package ua.antibyte.cinema.service.inject;
 
 import java.time.LocalDateTime;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
 import ua.antibyte.cinema.model.CinemaHall;
 import ua.antibyte.cinema.model.Movie;
 import ua.antibyte.cinema.model.MovieSession;
+import ua.antibyte.cinema.model.Role;
+import ua.antibyte.cinema.model.User;
 import ua.antibyte.cinema.service.CinemaHallService;
 import ua.antibyte.cinema.service.MovieService;
 import ua.antibyte.cinema.service.MovieSessionService;
-import ua.antibyte.cinema.service.security.AuthenticationService;
+import ua.antibyte.cinema.service.RoleService;
+import ua.antibyte.cinema.service.ShoppingCartService;
+import ua.antibyte.cinema.service.UserService;
 
-@RestController
-@RequestMapping("/inject")
+@Component
 public class Inject {
```

Comment:
```suggestion
public class DefaultDataInitializer/DefaultDataInjector {
```
class names are nouns not verbs